Framework/include/QualityControl/HistoMerger.h

```diff
@@ -0,0 +1,65 @@
+///
+/// \file   HistoMerger.h
+/// \author Piotr Konopka
+///
+
+#ifndef QUALITYCONTROL_HISTOMERGER_H
+#define QUALITYCONTROL_HISTOMERGER_H
+
+#include <vector>
+#include <memory>
+
+#include <TH1.h>
+#include <Framework/Task.h>
+#include <Headers/DataHeader.h>
+#include "Common/Timer.h"
+
+#include "QualityControl/MonitorObject.h"
+
+namespace o2 {
+namespace quality_control {
+namespace core {
+
+
+/// \brief A crude histogram merger for development purposes.
```

Comment:
Maybe just add the limitations. Something like "It handles a single histogram and it expects to receive updates as opposed to full objects".